CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [v0.0.38](http://github.com/ndustrialio/contxt-sdk-js/tree/v0.0.38) (2019-xx-xx)
+
+**Added**
+
+* Added Files module with the ability to perform read, download on Files stored in the Files API
```

Comment:
Can you list out which endpoints specifically?

Reply:
On it